### ISSUE TITLE
SMAPI Does Not Respect Console Color Schemes

### ISSUE THREAD

**Issue by TheConfectioner** [Pathoschild/SMAPI#942]
**Describe the bug**
I came across issue #707 while looking for a solution on how to improve the readability of logs in SMAPI console and then followed the suggestion to add `config.user.json`. I added the file and changed the contents to my liking, but smapi still used the old color scheme. I also added `SMAPI-config.json` in the Mods folder to make sure that the documentation was correct, but the color scheme also remained unchanged. I only realized the bug after forcing to change all text colors in the scheme (both in dark and light backgrounds) to white. I re-edited my configuration and realized that SMAPI only reads the color scheme for light backgrounds.

**To Reproduce**
1. add `smapi-internal/config.user.json` or `Mods/SMAPI-config.json` file with customized settings. the following is the initial configuration:
```
{
   "ConsoleColors": {
      "UseScheme": "None"
   }
}
```
3. launch smapi and see that the color scheme has not changed.
4. change the color settings with the following scheme:
```
{
    "ConsoleColors": {
        "UseScheme": "DarkBackground",

        "Schemes": {
            "DarkBackground": {
                "Trace": "White",
                "Debug": "White",
                "Info": "White",
                "Warn": "White",
                "Error": "White",
                "Alert": "White",
                "Success": "White"
            },
            "LightBackground": {
                "Trace": "Gray",
                "Debug": "Gray",
                "Info": "White",
                "Warn": "Yellow",
                "Error": "Red",
                "Alert": "Magenta",
                "Success": "Green"
            }
        }
    }
}
```
6. launch smapi and notice that now the colors have changed.

**Log file**
https://smapi.io/log/b5eb4419a1b844f8863f0d40409efbe8

**Screenshots**
The result using initial configuration:
![Firsrt result](https://github.com/Pathoschild/SMAPI/assets/114721084/f35bb94a-2832-4adc-beaa-49ad398839d0)
The result using later configuration:
![Second result](https://github.com/Pathoschild/SMAPI/assets/114721084/fd84c8f7-6053-46b7-97b2-54db63ca7c0e)

**Comment by nrobinson12** [comment]
I have tested this too; running on xterm. It is not respecting the json files.

**Comment by Pathoschild** [comment]
Fixed in the upcoming SMAPI 4.1.9. Thanks for reporting it!
